specifications.yml

```diff
@@ -41,16 +41,16 @@ paths:
       security:
         - oAuth2ClientCredentials: []
 
-  /application/customer/{customerApplicationId}:
+  /application/customer/{applicationId}:
     get:
       tags:
       - Application endpoints
-      summary: Get a customer application
-      description: Fetch a customer application
+      summary: Get a customer
```

Comment:
Wait why change this to simply get a customer, arent we fetching an application?